docs/docker.md

```diff
@@ -0,0 +1,65 @@
+## Setting up Docker
+
+Bison includes a `docker-compose.yml`, `Dockerfile` and `.dockerignore`
+
+The configuration is to run a postgres database locally. 
+
+If you are interested in running the entire app, please see the examples below.
+
+## Running the Database
+
+To run postgres with docker, please add the following information to your `env` & `env.local` file.
+```
+DATABASE_URL="postgresql://dev:dev@localhost:5432/dev?schema=public"
+```
+Run `docker-compose up -d` in your terminal. If you prefer not to run in detached mode, you can remove the `-d` flag.
+
+To shut down docker, run `docker-compose down`
+
+## Using NextJS & Postgres with Docker
```

Comment:
Just to be consistent with Next's official name, it's `Next.js`. Same throughout.